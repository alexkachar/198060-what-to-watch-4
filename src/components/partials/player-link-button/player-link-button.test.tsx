import * as React from 'react';
import * as renderer from 'react-test-renderer';
import {BrowserRouter} from 'react-router-dom';

import PlayerLinkButton from './player-link-button';

it(`PlayerLinkButton renders correctly`, () => {
  const tree = renderer
    .create(
        <BrowserRouter>
          <PlayerLinkButton
            id={`1`}
          />
        </BrowserRouter>, {
          createNodeMock: () => document.createElement(`div`)
        })
    .toJSON();
  expect(tree).toMatchSnapshot();
});
