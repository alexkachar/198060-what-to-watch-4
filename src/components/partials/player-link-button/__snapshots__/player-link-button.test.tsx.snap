// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayerLinkButton renders correctly 1`] = `
<a
  className="btn btn--play movie-card__button"
  href="/player/1"
  onClick={[Function]}
>
  <svg
    height={19}
    viewBox="0 0 21 19"
    width={21}
  >
    <use
      xlinkHref="#play-s"
    />
  </svg>
  <span>
    Play
  </span>
</a>
`;
